### ISSUE TITLE
Vintage is gold - install and usage combined with official release

### ISSUE THREAD

**Issue by Nickwiz** [Rafficer/linux-cli-community#352]
I am current on Ubuntu 18.04 LTS. Plan is to move to another flavor of Linux then Ubuntu, but both dread it and can't make up my mind for what to go for :P

I'm currently testing out ProtonVPN free version. So far so good. I installed the deb but as on 18.04 I can not use the GUI version, and the cli version is perhaps limited as well.

Found this project and cloned it, but as I have my setup working and running I'd rather not mess tings up so my first thought was to run it as a venv hence I did:

```sh
git clone https://github.com/Rafficer/linux-cli-community.git
cd linux-cli-community 
python3 -m venv .
bin/pip3 install protonvpn-cli
bin/protonvpn s

ProtonVPN now offers an official Linux app which includes a graphical user interface.
Visit https://protonvpn.com/support/official-linux-client to upgrade.
[!] There has been no profile initialized yet. Please run 'protonvpn init'.
```

So far so good, but now I wonder if this is a path I should continue on. Would `bin/protonvpn init` mess up the native installed VPN settings? Can I run it in a virtual environment like this? Is this repo behind the native due to the message given?

In short: A way to run this on older systems.

Guess this could be OK / good for many a system out there that is "stuck" on old releases for what ever reason. E.g. I have a laptop with a heavily modified OS + ancient architecture which I'd like to use this on if I get it to work.
